packages/frontend/src/components/DashboardTiles/TileBase/index.tsx

```diff
@@ -111,13 +111,19 @@ const TileBase = <T extends Dashboard['tiles'][number]>({
             bg="white"
             radius="sm"
             shadow={isEditMode ? 'xs' : undefined}
-            sx={(theme) => ({
-                overflow: 'unset',
-
-                border: isEditMode
-                    ? `1px dashed ${theme.colors.blue[5]}`
-                    : `1px solid ${theme.colors.gray[1]}`,
-            })}
+            sx={(theme) => {
+                let border = `1px solid ${theme.colors.gray[1]}`;
+                if (tabs && tabs.length > 1) {
+                    border = `1px solid ${theme.colors.gray[3]}`;
```

Comment:
@PriPatel can you confirm what gray we should use? ticket mentioned gray 2 but gray 3 is closer to the screenshot you provided.

Reply:
I think gray3 is correct! (looks good in the PR)